node/src/BaseClient.ts

```diff
@@ -4136,6 +4138,59 @@ export class BaseClient {
         return this.createWritePromise(createZIncrBy(key, increment, member));
     }
 
+    /**
+     * Iterates incrementally over a sorted set.
+     *
+     * See https://valkey.io/commands/zscan for more details.
+     *
+     * @param key - The key of the sorted set.
+     * @param cursor - The cursor that points to the next iteration of results. A value of "0" indicates the start of
+     *      the search.
+     * @param options - (Optional) The zscan options.
+     * @returns An `Array` of the `cursor` and the subset of the sorted set held by `key`.
+     *      The first element is always the `cursor` for the next iteration of results. `0` will be the `cursor`
+     *      returned on the last iteration of the sorted set. The second element is always an `Array` of the subset
+     *      of the sorted set held in `key`. The `Array` in the second element is always a flattened series of
+     *      `String` pairs, where the value is at even indices and the score is at odd indices.
+     *
+     * @example
+     * ```typescript
+     * // Assume "key" contains a sorted set with multiple members
+     * let resultCursor = "1";
+     * let newCursor = "0";
+     * let result = [];
+     *
+     * while (resultCursor !== "0") {
+     *      result = await client.zscan("key", newCursor, {
+     *          match: "*",
+     *          count: 5,
+     *      });
+     *      newCursor = result[0];
+     *      console.log("Cursor: ", newCursor);
+     *      console.log("Members: ", result[1]);
+     *
+     *      if (newCursor === "0") {
+     *          break;
+     *      }
+     *
+     *      resultCursor = newCursor;
+     * }
+     * // Cursor:  123
```

Comment:
Add a comment like: "Code produces the following output" or "... output like this"